Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -380,6 +380,45 @@ def create_publications(cve: dict) -> list:
     return publications
 
 
+def build_ids_params(ids: list) -> str:
+    """
+        Gets a list of IDs and return a string to use as a query param in the requests of exclusion entities.
+        For example: ['1234', '5678'] => '?ids=1234&ids=5678'
+
+        Args:
+            ids: List of exclusion IDs.
+        Returns:
+            String to use as a query param in the requests of exclusion.
+    """
+    params = f'?ids={ids[0]}'
+
+    if (ids_length := len(ids)) > 1:
+        for i in range(1, ids_length):
+            params += f'&ids={ids[i]}'
+
+    return params
```

Comment:
```suggestion
    return '?ids' + '&ids='.join(ids)
```
And if we do it do we need a function for one line ?